mars/tensor/base/tests/test_base.py

```diff
@@ -370,6 +370,13 @@ def testSplit(self):
         self.assertTrue(splits[2].flags['F_CONTIGUOUS'])
         self.assertFalse(splits[0].flags['C_CONTIGUOUS'])
 
+        for a in ((1,1,1,2,2,3), [1,1,1,2,2,3]):
+            splits = split(a, (3,5))
+            self.assertEqual(len(splits), 3)
+            np.testing.assert_array_equal(splits[0].execute(), (1,1,1))
```

Comment:
In our unittests, like `test_base.py` only test the expression part, which means, `.execute()` will not show up in these tests. 

The unittests for execution can be added to `test_base_execute.py` to verify the result.

Reply:
Here I think test the length is enough, no need to verify the result any further.